guide/matrix-vs-telegram.md

```diff
@@ -28,32 +28,40 @@ Despite recent studies, Telegram's use of its own MTProto protocol remains a deb
+| **Username** | Users are identified by phone number or username (if set up, 5~32 alphanumeric characters) to fellow users, and user IDs (around 9~10 digits) for programming purposes. A display name can be added (no limit). | Users are identified by their MXID (eg. `@alice:example.com`), composed of the username (must be ASCII characters, up to 255 characters) and the server name. A display name can be optionally added (up to ~65200 bytes)[^2]. |
 | Avatar | Static or animated; limit unknown. Can be zoomed; the returned avatar has a maximum definition of 640x640. | **See "Attachments" for limits.** Can be zoomed (at least in Element/SchildiChat), in which case the avatar will be shown in the uploaded definition. Animated avatars are supported and will be rendered (at least in Element/SchildiChat). |
-| Profile description | **Supported**. | Not supported. |
-| Room-specific nicknames | Not supported, though group admins can talk on behalf of the whole group. | **Supported** (`/myroomnick`). |
+| Profile description | **Supported**. | Will be supported using profile rooms. Not supported currently. |
+| Room-specific nicknames | Not supported, though group admins can talk on behalf of the whole group. | **Supported** (`/myroomnick`). Up to ~65200 bytes. [^2] |
 | Room-specific avatars | Not supported. | [**Supported**](../features/#attachments) (`/myroomavatar`). See "Attachments" for limits. |
 | 2FA | One-time token sent to another session. | Not required for login, but required (QR code, emoji verification, or Security Key) for viewing past encrypted messages. |
-| **Text messages** | Maximum 4096 characters. Supports Markdown. | **No maximum. [Supports Markdown and HTML.](../features/#text)** |
+| **Text messages** | Maximum 4096 characters. Supports Markdown. | **Up to ~65200 bytes (up to ~40770 bytes if a formatted message with plain text fallback sent).[^2] [Supports Markdown and HTML.](../features/#text)** |
 | Attachments | **Maximum 2 GB.** | Maximum 50~100 MB (for most homeservers; customizable if you run your own homeserver). |
 | Reactions | Very limited. Must be enabled by group admins in groups. | All unicode emotes and [text](../features/#reactions). |
 | Stickers | Up to 200 packs of 120 static or 50 animated each. | **Unlimited (static or animated) with setup.** See [here](../features/#stickers). |
-| Read receipts | Supported ambiguously. | **Supported.** |
+| Public read receipts | Supported ambiguously. | **Supported.** |
 | **Direct messages** | Not encrypted unless explicitly opted into secret chat, which cannot be carried across devices. VoIP is encrypted. | **Encrypted by default**, including VoIP. |
 | **Group chats** | You can join up to 500 groups and channels. | You can join an **unlimited** amount of rooms. |
 | VoIP in groups | Supported. | Not supported until early 2022. |
```

Comment:
Technically supported but only via Jitsi Widgets and Element clients are the only clients that have proper support for this. Expected to be replaced during 2022 with a better system that is a part of the matrix spec. 
```suggestion
| VoIP in groups | Supported. | Limited Support via Integration with Jitsi. Expected to be replaced by a better solution during 2022. |
```